discord/raw_models.py

```diff
@@ -192,6 +192,8 @@ class RawReactionActionEvent(_RawReprMixin):
         The guild ID where the reaction got added or removed, if applicable.
```

Comment:
Observe the api, there are more properties afaik

Reply:
hmmm not from what I can tell? 
```py
{
	'user_id': '242367619431268353',
	'message_id': '1079414819364548608',
	'member': {...},
	'emoji': {
		'name': '⭐',
		'id': None
	},
	'channel_id': '930883593658589194',
	'burst': False,
	'guild_id': '282550444923027457',
	'me': False
}
``` 
unless there's more to come...

Reply:
oh uhm, nvm, it's not live yet in the bots api what i thought of xD

Reply:
Ok so I've found the additional properties. Gonna dm them to you later.